### ISSUE TITLE
ABINet to tensorRT

### ISSUE THREAD

**Issue by Hegelim** [open-mmlab/mmocr#1124]
I have 2 quick questions:

1. From this page https://mmocr.readthedocs.io/en/latest/deployment.html#list-of-supported-models-exportable-to-onnx
Looks like there's currently no support to convert ABINEt -> ONNX -> tensorRT, right? If so, when is it expected to be supported? 

2. From here https://github.com/NVIDIA-AI-IOT/torch2trt
I am wondering if we can call torch2trt directly to convert pretrained ABINet to tensorRT? 

Thanks

**Comment by gaotongxiao** [comment]
cc @AllentDan. These deployment-related issues can be raised to MMDeploy for faster and accurate response.

**Comment by AllentDan** [comment]
Hi, @Hegelim. Deployment stuff is now moved to [MMDeploy](https://github.com/open-mmlab/mmdeploy) for better supports and it will be deprecated in MMOCR. For your questions:
1. ABINet is not in the supported model list
2. torch2trt is a method converting to TensorRT. But we chose torch->ONNX->TensorRT as ONNX plays the role of intermediate representation here and it can be converted to other backends like OpenVINO or ncnn.
